src/main/java/com/taf/business/driver/DriverManager.java

```diff
@@ -0,0 +1,31 @@
+package com.taf.business.driver;
+
+import org.openqa.selenium.WebDriver;
+
+public class DriverManager {
+
+    private static final ThreadLocal<WebDriver> containerWebDriver = ThreadLocal.withInitial(() -> null);
+    private static DriverManager instance;
+
+    private DriverManager() {
+    }
+
+    public static DriverManager getInstance() {
```

Comment:
I didn't find usages of this method, most likely it can be omitted along with `instance` variable

Reply:
It was deleted, thanks.